java/arcs/core/data/Capability.kt

```diff
@@ -39,6 +39,17 @@ sealed class Capability(val tag: String) {
         }
     }
 
+    fun getRealTag(): String {
```

Comment:
KDoc for this? (What's a "real tag"?)

Reply:
done (i'm not a huge fan of this name, but couldn't think of anything better).